src/libsync/checksums.cpp

```diff
@@ -120,6 +132,12 @@ QByteArray contentChecksumType()
     return type;
 }
 
+QList<QByteArray> collisionSafeHashes()
+{
+    static QList<QByteArray> list = { "SHA1" };
```

Comment:
Well well well....  haven't you heard the news about sha1?
But yeah, since we don't support anything better anyway...